examples/pong.qasm

```diff
@@ -4,12 +4,12 @@
  * 2 PPPMMMMMMPPP
  */
 
-gatecal pong(amp) q0 [duration] {
+defcal pong(amp, duration) %q0 {
     play d0, gaussian(amp, duration)
 }
 
-gatecal pong_cx(amp) q0, q1, q2{
-    barrier q0, q1, q2;
+gatecal pong_cx(amp) %q0, %q1, %q2 {
+    barrier %q0, %q1, %q2;
     cross-res(pi/4) 0, 1
```

Comment:
@lcapelluto @stevenheidel we should make sure to update this example when the pulse grammar is filled in.